src/main/java/com/google/devtools/build/lib/skyframe/BzlLoadFunction.java

```diff
@@ -586,7 +586,8 @@ private BzlLoadValue computeInternal(
   private StarlarkBuiltinsValue getBuiltins(
       BzlLoadValue.Key key, Environment env, @Nullable InliningState inliningState)
       throws BzlLoadFailedException, InterruptedException {
-    if (!(key instanceof BzlLoadValue.KeyForBuild)) {
+    if (!(key instanceof BzlLoadValue.KeyForBuild)
```

Comment:
There are 4 keys (also for builtins and for bzlmod). Does this condition work if it's changed to `key.isBuiltins()`? That would be slightly more simple.